docs/kjar/README.md

```diff
@@ -23,18 +23,29 @@ Accordingly to your used module change
 [springboot module](/springboot/kubernetes/deployment.yaml)
 or
 [jdkhttp module](/jdkhttp/kubernetes/deployment.yaml)
+
+add the runAsAuser id and add the env vars
+change the value from the example with the correct runAsAUser ID, your KJARGAV and your MAVEN_MIRROR_URL with your nexus,  
+the MAVEN_LOCAL_REPO and MAVEN_SETTINGS_XML are the values with the default configuration in the docker container and UPDATABLEKJAR with value "true"
+to enable the update of the kjar
 ```yaml
-containers:
+      securityContext:
+        runAsUser: 1000170000
```

Comment:
Please change to "<id_user>" placeholder.

Reply:
ok